### PR TITLE
Add documentation link for installing on IBM Cloud.

### PR DESCRIPTION
**Pull request by KissConsult**
Based on #16450
Step by step tutorial for installing on IBM Cloud

Signed-off-by: Isaac Bennetch <bennetch@gmail.com>

### CODE REVIEW

**Comment by KissConsult** [comment]
@ibennetch @williamdes 
Hi , can i get a reviewer or assignee ? @ibennetch I made the pull request from your repository based on the changes you made . 

Thanks!

File: doc/setup.rst [hunk 1]
```diff
@@ -484,6 +484,12 @@ configuration:
           - PMA_PASSWORD=
           - PMA_ABSOLUTE_URI=http://example.com/phpmyadmin/
 
+IBM Cloud
++++++++++
+
+One of our users has created a helpful guide for installing phpMyAdmin on the
+`IBM Cloud platform <https://github.com/KissConsult/phpmyadmin_tutorial>`_.
```
Comment (Guileas) [inline doc/setup.rst:491]: Replace 
`IBM Cloud platform <https://github.com/KissConsult/phpmyadmin_tutorial>_.`

By 
`IBM Cloud platform <https://github.com/KissConsult/phpmyadmin_tutorial/blob/main/README.md>`

Comment (ibennetch) [reply doc/setup.rst]: Good suggestion @Guileas, I will make that change in my branch.

**Review by williamdes — COMMENTED** [review]
Perfect, merged !

**Pull request merged**